playground/backend/internal/environment/environment_service_test.go

```diff
@@ -50,11 +52,18 @@ func setup() error {
 		return err
 	}
 	os.Clearenv()
+
+	executorConfig = NewExecutorConfig(
+		"javac", "java", "java",
+		[]string{"-d", "bin", "-classpath", jarsPath},
+		[]string{"-cp", "bin:" + jarsPath},
+		[]string{"-cp", "bin:" + jarsPath, "JUnit"},
+	)
 	return nil
 }
 
 func teardown() {
-	err := os.RemoveAll(configFolderName)
+	err := os.Remove(configFolderName)
```

Comment:
If I understand correctly `configFolderName` is the name of the folder which contains some files. So if we want to remove this folder with all included files we need to call `RemoveAll()` instead of `Remove()`

Reply:
you right, it is done by mistake